features/main/always_identifier.feature

```diff
@@ -0,0 +1,117 @@
+    And I send a "POST" request to "/always_identifier_dummies" with body:
+    """
+    {
+      "children": [
+          "/always_identifier_dummies/1"
+      ],
+      "related": [
+          "/always_identifier_dummies/1"
+      ],
+      "parent": "/always_identifier_dummies/1"
+    }
+    """
+    And the JSON should be equal to:
+    """
+    {
+        "@context": "/contexts/AlwaysIdentifierDummy",
+        "@id": "/always_identifier_dummies/2",
+        "@type": "AlwaysIdentifierDummy",
+        "children": [
+            "/always_identifier_dummies/1"
+        ],
+        "related": [
+            {
+                "@id": "\/always_identifier_dummies\/1",
+                "@type": "AlwaysIdentifierDummy",
+                "children": [],
+                "related": [],
+                "parent": null
+            }
+        ],
+        "parent": "/always_identifier_dummies/1"
+    }
+    """
+
+    Scenario: Check the json formatted response
+      When I add "Accept" header equal to "application/json"
+      And I add "Content-Type" header equal to "application/json"
+      And I send a "GET" request to "/always_identifier_dummies/2"
+      Then the JSON should be equal to:
+      """
+      {
+          "children": [
+              "\/always_identifier_dummies\/1"
+          ],
+          "related": [
+              {
+                  "children": [],
+                  "related": [],
+                  "parent": null
+              }
+          ],
+          "parent": "\/always_identifier_dummies\/1"
+      }
+      """
+
+  Scenario: Check the hal formatted response
+    When I add "Accept" header equal to "application/hal+json"
+    And I add "Content-Type" header equal to "application/hal+json"
+    And I send a "GET" request to "/always_identifier_dummies/2"
+    Then the JSON should be equal to:
+      """
+      {
+          "_links": {
+              "self": {
+                  "href": "\/always_identifier_dummies\/2"
+              },
+              "children": [
+                  {
+                      "href": "\/always_identifier_dummies\/1"
+                  }
+              ],
+              "related": [
+                  {
+                      "href": "\/always_identifier_dummies\/1"
+                  }
+              ],
+              "parent": {
+                  "href": "\/always_identifier_dummies\/1"
+              }
+          },
+          "_embedded": {
+              "children": [
+                  "\/always_identifier_dummies\/1"
+              ],
+              "related": [
+                  {
+                      "_links": {
+                          "self": {
+                              "href": "\/always_identifier_dummies\/1"
+                          }
+                      }
+                  }
+              ],
+              "parent": "\/always_identifier_dummies\/1"
+          }
+      }
+      """
+
+  @dropSchema
+  Scenario: Delete a resource
```

Comment:
This is not necessary (this is not related to this feature and already tested). `@dropSchema` will already destroy the database.